proposals/record-structs.md

```diff
@@ -298,4 +304,5 @@ See https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-10-0
 - confirm that we want to disallow members named "Clone".
 - why did we disallow unsafe types in instance fields in records? I assume we also want to disallow in record structs.
 - `with` on generics? (may affect the design for record structs)
+- confirm we won't allow `record ref struct`.
```

Comment:
Probably worth adding a note of the elements we talked about in the team meeting today around `IEquatable<T>` and ref fields.